### PR TITLE
Xeora Language implementation

### PR DESCRIPTION
**Pull request by freakmaxi**
Hello,

This is the Xeora Web Development Framework grammar implementation. 

http://www.xeora.org
https://www.github.com/xeora

Thanks...

### CODE REVIEW

File: components/prism-xeora.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+						'tag': {
+							pattern: /#\d/
+						}
+					}
+				}
+			},
+			alias: 'function'
+		},
+		'directive-block-open': {
+			pattern: /\$\w+:{|\$\w(#\d+(\+)?)?(\[[\.\w\-]+])?:[\.\w\-]+:{(![A-Z]+)?/,
```
Comment (Golmote) [inline components/prism-xeora.js]: You can remove the parentheses around `(\+)?`
You can remove the escapes in `[\.\w\-]` -> `[.\w-]`
Please make the groups non-capturing.

File: components/prism-xeora.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+							pattern: /!/
+						}
+					},
+					alias: 'keyword'
+				}
+			},
+			alias: 'function'
+		},
+		'directive-block-separator': {
+			pattern: /}:[\.\w\-]+:{/,
```
Comment (Golmote) [inline components/prism-xeora.js]: `[\.\w\-]` -> `[.\w-]`

File: components/prism-xeora.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+			inside: {
+				'punctuation': {
+					pattern: /}|:|{/
```
Comment (Golmote) [inline components/prism-xeora.js]: Can be simplified to `[:{}]`

File: components/prism-xeora.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+				}
+			},
+			alias: 'function'
+		},
+		'directive-block-close': {
+			pattern: /}:[\.\w\-]+\$/,
```
Comment (Golmote) [inline components/prism-xeora.js]: Same comment as previously.

File: components/prism-xeora.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+			inside: {
+				'punctuation': {
+					pattern: /}|:|\$/
```
Comment (Golmote) [inline components/prism-xeora.js]: Can be simplified to `[}:$]`

**Comment by Golmote** [comment]
@freakmaxi Can you please fix the example so that it's like [all the others](https://github.com/PrismJS/prism/tree/gh-pages/examples)? (Add a title, explicitely state which class to use, add subtitles listing the main features of the component with examples in code blocks, etc.)

**Comment by freakmaxi** [comment]
Oh my god, I'm so sorry. I didn't realize that there is a format for examples. 😞 I'm fixing it immediately.

File: components.js [hunk 6]
```diff
@@ -598,6 +598,11 @@ var components = {
 			"title": "Xojo (REALbasic)",
 			"owner": "Golmote"
 		},
+		"xeora": {
+			"title": "Xeora",
+			"require": "markup",
+			"owner": "xeora"
```
Comment (Golmote) [inline components.js]: Please set the owner to you Github username `"freakmaxi"`.

Once this is changed I think we're good to go!

**Comment by Golmote** [comment]
Thanks for contributing!

**Comment by freakmaxi** [comment]
I thank you for your interest and cooperation. That was a pleasure for me 👍

**Pull request merged**